src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -151,38 +151,59 @@ await _methodologyVersionRepository.GetLatestPublishedVersion(methodology.Id))
                 .OnSuccess(BuildMethodologyVersionViewModel);
         }
 
-        public async Task<Either<ActionResult, List<MethodologyVersionSummaryViewModel>>> ListLatestMethodologyVersions(Guid publicationId)
+        public async Task<Either<ActionResult, List<MethodologyVersionSummaryViewModel>>>
+            ListLatestMethodologyVersions(
+                Guid publicationId,
+                bool isPrerelease = false)
         {
             return await _persistenceHelper.CheckEntityExists<Publication>(publicationId,
-                    q => q.Include(p => p.Methodologies)
-                        .ThenInclude(p => p.Methodology)
-                        .ThenInclude(p => p.Versions))
+                    q => q.Include(publication => publication.Methodologies)
+                        .ThenInclude(publicationMethodology => publicationMethodology.Methodology)
+                        .ThenInclude(methodology => methodology.Versions)
+                        .ThenInclude(versions => versions.PreviousVersion))
                 .OnSuccess(publication => _userService.CheckCanViewPublication(publication))
                 .OnSuccess(async publication =>
                 {
                     return await publication.Methodologies
                         .ToAsyncEnumerable()
                         .SelectAwait(async publicationMethodology =>
                         {
-                            var latestVersion = publicationMethodology.Methodology.LatestVersion();
+                            var methodologyVersion = publicationMethodology.Methodology.LatestVersion();
+
+                            if (isPrerelease && methodologyVersion.Status != MethodologyApprovalStatus.Approved)
+                            {
+                                // Get latest approved versions
+                                if (methodologyVersion.PreviousVersion != null)
+                                {
+                                    // If there is a previous version, it must be approved, because cannot
+                                    // create an amendment for an unpublished version
+                                    methodologyVersion = methodologyVersion.PreviousVersion;
+                                }
+                                else
+                                {
+                                    return null;
+                                }
```

Comment:
Inverting this would be shorter and easier to read in my opinion

```
  // Get latest approved versions
  if (methodologyVersion.PreviousVersion == null)
  {
    return null;
  }

  // Use the previous version which must be approved, because cannot
  // create an amendment for an unpublished version
  methodologyVersion = methodologyVersion.PreviousVersion
```